how-to/customize-workspace/client/src/modules/actions/app-definition/index.ts

```diff
@@ -1,3 +1,6 @@
+import type { ModuleImplementation, ModuleTypes } from "../../../module-shapes";
 import { AppDefinitionActions } from "./actions";
 
-export const actions = new AppDefinitionActions();
+export const entryPoints: { [type in ModuleTypes]?: ModuleImplementation } = {
```

Comment:
guessing it is called entry points because the new module interface needs it?

Reply:
Yes, its a more generic term as a module can contain multiple types.